README.md

```diff
@@ -12,14 +11,20 @@ Popper is a convention and CLI tool for generating reproducible papers. This rep
     convention and the CLI tool.
   * Contents of our [landing page](http://falsifiable.us) ([`gh-pages`](https://github.com/systemslab/popper/tree/gh-pages) branch).
 
+
+#### 5 minute screencast demo of the CLI
+  
+  [![asciicast](https://asciinema.org/a/xzEYfI3U4H5CWdmjBe0O2CAw8.png)](https://asciinema.org/a/xzEYfI3U4H5CWdmjBe0O2CAw8)
```

Comment:
Can the size of the image be reduced?

Reply:
It might be better to leave it out as we had it before. Or to reduce the size of it and put it to the end.